Remove duplicate quotes

omackenzie opened the pull request:
Fixes #1361 
I wrote a quick Python script to remove any duplicate quotes. It found and removed 1 in code_rust, 4 in english and 2 in turkish.

omackenzie commented:
IDs of duplicate quotes are:

In code_rust.json:
- 50 and 52

In english.json:
- 4963 and 5448
- 5032 and 5450
- 261 and 5470
- 5160 and 5474

In turkish.json
- 8 and 12
- 15 and 19

Miodec commented on static/quotes/code_rust.json:
```diff
@@ -1,1673 +1,1667 @@
+    {
+      "id": 35,
+      "length": 13,
+      "source": "Assign to string the japanese word ネコ - programming-idioms.org",
+      "text": "let s = \"ネコ\";"
```
Please revert this change

Miodec commented on static/quotes/code_rust.json:
```diff
@@ -1,1673 +1,1667 @@
+      "id": 52,
+      "length": 100,
+      "source": "First-class function : generic composition - programming-idioms.org",
+      "text": "fn compose<A, B, C>(f: impl Fn(A) -> B, g: impl Fn(B) -> C) -> impl Fn(A) -> C {\n\tmove |x| g(f(x))\n}"
```
As far as i can tell this was not a duplicate

omackenzie commented:
I think I've fixed the issues. As for the rust quote, yes they technically have different titles, but the text is the exact same. I reverted the change but I think it's something to look at.

Miodec commented:
The pr is still full of whitespace changes 
![image](https://user-images.githubusercontent.com/13181393/117452233-6c4f4e00-af3b-11eb-9cae-5adcb0f7016a.png)

omackenzie commented:
Should be fixed now. My script changed the line endings from LF to CRLF.

Miodec commented:
Nice.
